CATALOG.md

```diff
@@ -187,3 +187,63 @@ a[href='/download'] {
   display: none;
 }
 ```
+
+
+### Notion Pages with interactive buttons
+
```

Comment:
Can you add here a two-liner for context aimed at helping a novice passerby? What's broken originally, a high-level overview of how it's fixed, and who should use this.

Reply:
Sure thing! I will add it and update the PR. Cheers!